src/Tools/Performance Toolkit/App/src/BCPTLine.Codeunit.al

```diff
@@ -164,7 +164,11 @@ codeunit 149005 "BCPT Line"
     var
         BCPTLogEntry: Record "BCPT Log Entry";
         BCPTRoleWrapperImpl: Codeunit "BCPT Role Wrapper"; // single instance
+        ValuesAreChanged: Boolean;
+        BCPTTestSuite: Codeunit "BCPT Test Suite";
     begin
+        BCPTTestSuite.OnBeforeBCPTLineAddLogEntry(BCPTLine."BCPT Code", BCPTLine."Codeunit ID", BCPTLine.Description, Operation, ExecutionSuccess, Message, Operation, ExecutionSuccess, Message);
```

Comment:
No, not all.

The discussion is that I want an event to change some of the parameters that comes into function AddLogEntry.
But Gert wanted a way for other subscribers to know if the value has been changed hence why we added the "Original value" parameter.
Example, the operation and the "Orig. Operation" parameters.
Subscribers can now check if the Operation parameter has already been changed by other subscribers by doing following code:
`If "Orig. Operation" <> Operation then ....`
![image](https://github.com/microsoft/BCApps/assets/34970096/acbb64c0-8d29-46e8-b285-f493f82e0d09)

Reply:
Wouldn't the changes I proposed resolve the concerns? Subscribers have access to original and modified content and content is also stored in the BCPT Log Entry table for future auditing if needed.

Reply:
Well, yes. 
I just don't see the need for it to be stored in table. But overall it is a more correct way if someone would be interested in that audit.

Reply:
Yes please. It is important to log the original result as the subscriber can completely overwrite the origin result without a trace:-).

Reply:
@ChethanT 
I'm not sure about this though?
```
if (Operation <> ModifiedOperation) or (ExecutionSuccess <> ModifiedExecutionSuccess) or (Message <> ModifiedMessage) then
EntryWasModified := true;

```
Since we are not using `EntryWasModified`.